app/src/main/java/org/mifos/mobile/ui/activities/MainActivity.kt

```diff
@@ -0,0 +1,77 @@
+package org.mifos.mobile.ui.activities
+
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.fragment.app.Fragment
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import org.mifos.mobile.R
+import org.mifos.mobile.ui.activities.base.BaseActivity
+import org.mifos.mobile.ui.fragments.AccountFragment
+import org.mifos.mobile.ui.fragments.NewHomeFragment
+import org.mifos.mobile.ui.fragments.ProfileFragment
+import org.mifos.mobile.ui.fragments.TransferFragment
+
+class MainActivity : BaseActivity(), BottomNavigationView.OnNavigationItemSelectedListener {
+
+    private var bottomNavigationView: BottomNavigationView? = null
+    private var menuItem = -1
```

Comment:
do we need menuItem at class level?

Reply:
No, we don't need this variable. I'll remove it. Sorry for the confusion.

Reply:
Done.